RandomCollectionExample.java

```diff
@@ -0,0 +1,37 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class RandomCollectionExample {
+    public static void main(String[] args) {
+        System.out.println(getRandomStrings(10));
+    }
+
+    public static List<String> getRandomStrings(int amount) {
+        List<String> myList = new ArrayList<String>();
+        int min = 6;
+        int max = 15;
+        int randomLength;
+        for (int i = 0; i < amount; i++) {
+            randomLength = randomChar(min, max);

```

Comment:
randomLength is **int** type
randomChar method returns **char** type
you don't need separate method to define length, you can do this with one line inside here like:
int randomLength = ThreadLocalRandom.current().nextInt(min, max + 1);

Reply:
d17c625